README.md

```diff
@@ -209,3 +209,16 @@ By default, the setup wizard (Jenkins >= 2.0) is skipped when using `hpi:run`. I
 
     mvn -Dhudson.Main.development=false hpi:run
 
+## Jenkins Core BOM
+
+Since version 2.195, Jenkins provides a [Maven Bill Of Materials (BOM)](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies)
+that centrally defines versions of various libraries used by Jenkins Core.
+The default behaviour of `plugin-pom` is to _not_ use the BOM, but when jenkins.version>=2.195 (and plugin-pom>=4.0) you can switch on Jenkins BOM support by setting the Maven property `use-jenkins-bom`.
```

Comment:
nit: suggest to use `.mvn/maven.config` for adding `-Duse-jenkins-bom`

Reply:
Agreed, its the preferred way and what the docs describe - was just keeping it brief here and referencing the docs for details.